source/admin_manual/list_of_events.rst

```diff
@@ -200,15 +200,26 @@ auth_client_cache_flush_finished
 Common fields
 -------------
 
+.. versionadded:: v2.3.13
+
 These fields are common for the rest of the authentication client events.
 
 +--------------------+-----------------------------------------------------------+
 | Field             | Description                                                |
 +===================+============================================================+
+| mechanism         | Name of used SASL mechanism (e.g. PLAIN)                   |
++-------------------+------------------------------------------------------------+
 | service           | Service name, such as imap3, pop3, lmtp.                   |
 +-------------------+------------------------------------------------------------+
+| transport         | Transport security indicator (insecure, TLS, trusted)      |
++-------------------+------------------------------------------------------------+
 | session           | Session identifier.                                        |
 +-------------------+------------------------------------------------------------+
+| certificate_user  | Username from certificate                                  |
++-------------------+------------------------------------------------------------+
+| client_id         | Expands to client ID request as IMAP arglist.              |
+|                   | Needs imap_id_retain=yes                                   |
```

Comment:
```
:ref:`setting-imap_id_retain`
```

Reply:
This was already fixed & merged today in another pull request.

Reply:
Oh, actually not, that was in variables page.

Reply:
Same issue with the existing client_id in this page, in the authentication server section. Should fix as well.